feature/experimental/bgp/ate_tests/bgp_2byte_4byte_asn/bgp_2byte_4byte_asn_test.go

```diff
@@ -0,0 +1,248 @@
+/*Here is the corrected code to establish eBGP IPv4 session between ATE-port1 and DUT-1:
```

Comment:
Replace this with the Google copyright header.

Reply:
Addressed